setup.py

```diff
@@ -46,16 +47,17 @@ def remove_bdist_rpm_source_file():
 shared_setup.remove_extra_bdist_rpm_files = remove_bdist_rpm_source_file
 shared_setup.SHARED_TARGET.update({
     'url': 'https://github.com/hpcugent/vsc-base',
-    'download_url': 'https://github.com/hpcugent/vsc-base'
+    'download_url': 'https://github.com/hpcugent/vsc-base',
+    'zip_safe': True,
 })
 
 
 PACKAGE = {
     'name': 'vsc-base',
-    'version': '2.2.6',
+    'version': '2.3.0',
```

Comment:
no, that doesn't work in rpm's, 2.2.7 is what I mean. 
It's fine for now, until we have an official version numbering policy I guess.